sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/SyntheticDataPubSubPublisher.java

```diff
@@ -73,6 +80,11 @@
     String getInsertionPipelineTopic();
 
     void setInsertionPipelineTopic(String topic);
+
+    @Description("Kafka server address (optional)")
```

Comment:
I would keep the info about this being optional only in documentation. Lack of `@Validation.Required` is enough. 

Usually when something is not required it has `@Default.` value. But as long as we treat this pipeline option as condition whether we use Kafka or PubSub, then it can leave it as it is now or change to default value `""` and then change the condition in the 98 line. WDYT?

Reply:
That's a good idea, done.